pkgs/development/python-modules/remctl/default.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, buildPythonPackage, isPy3k, pkgs, pytest, pytestrunner, typing }:
```

Comment:
```suggestion
{ lib, buildPythonPackage, isPy2k, remctl, pytestCheckHook, pytestrunner, typing }:
```

Reply:
The `pkgs.remctl -> remctl` change will cause infinite recursion since the python package is also named `remctl`. We have to explicitly refer to the package from `all-packages.nix`.

Reply:
It shouldn't cause on with the change in https://github.com/NixOS/nixpkgs/pull/110336#discussion_r561884997.

Reply:
Didn't notice that. Thanks!